tools/config-generator/customjobs.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Although custom jobs are not generated by this generator, certain testgrid
+// configs are needed for certain custom jobs
+
+package main
+
+func addCustomJobsTestgrid() {
+	var (
+		jobnames = []string{
+			"ci-knative-backup-artifacts",
```

Comment:
I'm wondering if we can add a unit test to make sure these Prow jobs exist.
We don't necessarily need to do it in this PR, but adding it will definitely make this part less brittle.

Reply:
Added one